Dockerfile

```diff
@@ -232,7 +234,8 @@ COPY server/Makefile server/Makefile
 RUN cd server && \
     make gen-server && \
     pip install -r requirements_cuda.txt && \
-    pip install ".[bnb, accelerate, quantize, peft, outlines]" --no-cache-dir
+    pip install ".[bnb, accelerate, quantize, peft, outlines]" --no-cache-dir && \
+    pip install nvidia-nccl-cu12==2.22.3
```

Comment:
Would have liked to use pyproject.toml for that, but poetry disapproves of conflict handling https://github.com/python-poetry/poetry/issues/697#issuecomment-470431668